devnet/singularity/cluster_config.json

```diff
@@ -39,7 +41,7 @@
         "DIFFICULTY": 10000000000000,
         "NONCE": 0
       },
-      "COINBASE_ADDRESS": "000000000000000000000000000000000000000000000000",
+      "COINBASE_ADDRESS": "123400000000000000000000000000000000000000000000",
```

Comment:
this will be a consensus change for our current testnet. how about `devnet/singularity/cluster_config_v2.json`?